spi/src/main/java/org/wildfly/swarm/spi/api/SwarmProperties.java

```diff
@@ -58,6 +58,11 @@
     String HTTPS_PORT = "swarm.https.port";
 
     /**
+     * If true, generates a self-signed certificate for development purposes
+     */
+    String HTTPS_GENERATE_SELF_SIGNED_CERTIFICATE = "swarm.https.generate_self_cert";
```

Comment:
Should this use all dots instead of hyphens?

Following the same pattern as `swarm.ds.connection.url`